docs/api.md

```diff
@@ -288,6 +289,8 @@ puppeteer.launch().then(async browser => {
   await browser2.close();
 });
 ```
+#### event: 'disconnected'
+emitted whenever the connection is closed `Browser.close() Browser.disconnect()`.
```

Comment:
Let's do the following wording:

> Emitted when puppeteer gets disconnected from the browser instance. This might happen because one of the following:
> - browser closed or crashed
> - `browser.disconnect` method was called